1.1.0 Changes

cs-abhishek-shukla opened the pull request:
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots

#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -17,11 +17,39 @@
+        <td>A playbook to return the JSON data to be rendered in the grid view.</td>
+    </tr>
+    <tr>
+        <td>Enable Grid Action Buttons</td>
```
"Enable to Grid Action button" is changed to "Grid Action Buttons"

cs-abhishek-shukla replied:
Changed

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -3,44 +3,82 @@
 
 - The following screenshot shows sample contents of the `grid_data` and `grid_columns` variable
 
     ![](./res/playbook_output.png)
```
The screenshot is not visible in file view.

cs-abhishek-shukla replied:
Removed.

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -3,44 +3,82 @@
+
+6. Select the checkbox **Action Buttons (No Record Selection)**.
+
+    ![](./res/edit-view-03.png)
```
Wrong screenshot attached, please check.

cs-abhishek-shukla replied:
Removed

cs-dev-chaudhary commented:
Approved

cs-dev-chaudhary commented on README.md:
```diff
@@ -10,11 +10,17 @@
 
 - **Applicable**: View Panel and Listing View
 
+- [Release Notes](./widget/release_notes.md)
```
Can we change the compatibility to 7.4.3 and later. As we have only tested from 7.4.3 this time.

cs-abhishek-shukla replied:
Done

cs-abhishek-shukla committed:
1.1.0 Changes
- Added full screen images
- Rewrote setup, readme, and Usage
- Annotated screenshots
- Added release notes
- included feedback
